src/writers.jl

```diff
@@ -175,7 +175,20 @@ end
 # LP File Writer
 # We use the formatting defined at:
 #   http://lpsolve.sourceforge.net/5.0/CPLEX-format.htm
-function writeLP(m::Model, fname::AbstractString)
+
+varname_generic(m::Model, col::Integer) = "VAR$(col)"
+
+function varname_given(m::Model, col::Integer)
+    # TODO: deal with non-ascii characters?
+    name = getname(m, col)
```

Comment:
I kind of want it to not fail, because I use greek letters a lot. So even if solvers can't read those `lp` files, I would use them for debugging.